src/providers/hana/CMakeLists.txt

```diff
@@ -0,0 +1,117 @@
+FIND_PACKAGE(ODBCCPP REQUIRED)
+
+########################################################
+# Files
+
+SET (HANA_SRCS
+  qgshanacolumntypethread.cpp
+  qgshanaconnection.cpp
+  qgshanaconnectionpool.cpp
+  qgshanaconnectionstringbuilder.cpp
+  qgshanadriver.cpp
+  qgshanaexpressioncompiler.cpp
+  qgshanafeatureiterator.cpp
+  qgshanaprimarykeys.cpp
+  qgshanaprovider.cpp
+  qgshanaproviderconnection.cpp
+  qgshanaresultset.cpp
+  qgshanasettings.cpp
+  qgshanatablemodel.cpp
+  qgshanacrsutils.cpp
+  qgshanautils.cpp
+)
+
+IF (WITH_GUI)
+  SET(HANA_SRCS ${HANA_SRCS}
+    qgshanadataitems.cpp
+    qgshanadataitemguiprovider.cpp
+    qgshananewconnection.cpp
+    qgshanaprovidergui.cpp
+    qgshanasourceselect.cpp
+  )
+ENDIF ()
+
+SET(HANA_HDRS
+  qgshanadriver.h
+  qgshanaexception.h
+  qgshanaexpressioncompiler.h
+  qgshanaproviderconnection.h
+  qgshanaresultset.h
+)
+
+########################################################
+# Build
+
+QT5_WRAP_CPP(HANA_MOC_SRCS ${HANA_MOC_HDRS})
```

Comment:
HANA_MOC_HDRS is not used anywhere -- this line has no effect

Reply:
Done in e5e7b31